src/main.c

```diff
@@ -383,29 +467,28 @@ bool handleLayer3SpecialCases(KBDLLHOOKSTRUCT keyInfo)
 			return true;
 		case 20:
 			sendChar(L'^', keyInfo);
-			keybd_event(VK_SPACE, 0x39, 0, 0);
+			commitDeadKey(keyInfo);
```

Comment:
Just a side note: I just noticed that I only get spaces instead of ^ in Neovim (same for backticks). But this also occurs with the official release. So not caused by your changes.